### ISSUE TITLE
Any plans to add a mod to Modrinth?

### ISSUE THREAD

**Issue by Alter73** [Elite-Modding-Team/MoCreaturesExtended#185]
Hi! I just make quite a lot of modpacks in the Modrinth launcher and a lot of mods have moved there. It is also possible because the service is developed only for Minecraft. It would be much more convenient if Mo' Creatures Extended was also in Modrinth, otherwise I have to download it separately for each modpack and check for its updates, but this is very inconvenient when creating modpacks.

**Comment by ACGaming** [comment]
We'll keep this in mind!

**Comment by Kylas21** [comment]
Yes, I'm also really looking forward to the appearance of this mod in Modrinth. And if the original mod is also uploaded, it will be very cool.
